docs/docs/guide/04-blog/00-express.md

```diff
@@ -1,10 +1,9 @@
 ---
-sidebar_position: 0
 description: Learn module basics by writing and reading blog posts to your chain.
-slug: /guide/blog
+title: Express
 ---
 
-# Build a blog
+# Build a blog with a single command
```

Comment:
there's actually more than one command :thinking: ...

Reply:
I might need to choose different phrasing for this. I meant it's one command for the blog app logic.